routes/post.go

```diff
@@ -377,6 +381,10 @@ func (fes *APIServer) GetPostEntriesByDESOAfterTimePaginated(readerPK []byte,
 		return nil, nil, fmt.Errorf("GetPostEntriesByDESO: Cannot fetch posts by deso more than an hour back")
 	}
 
+	if onlyNFTs && onlyPosts {
```

Comment:
this one is fine here since the only that happens above is also a simple validation